README.md

```diff
@@ -73,26 +78,92 @@ yarn deploy-local
       lastResultsHash
       evidenceHash
       proposerAddress
+      hash
     }
     evidence {
+      id
       evidence {
+        duplicateVoteEvidence {
```

Comment:
Is this added because there was a change in the protos ?

Reply:
that was in protos from the beginning

Reply:
i see duplicateVoteEvidence in previous version, sorry it's now in other place because I used alphabetical order